chromium_src/chrome/browser/flag_descriptions.cc

```diff
@@ -39,6 +39,9 @@ const char kBraveAdblockCspRulesDescription[] =
 const char kBraveDomainBlockName[] = "Enable domain blocking";
 const char kBraveDomainBlockDescription[] =
     "Enable support for blocking domains with an interstitial page";
+const char kBraveDebounceName[] = "Enable debouncing";
+const char kBraveDebounceDescription[] =
+    "Enable support for skipping certain tracking URLs.";
```

Comment:
Since this also includes the query string filter, the description should mention that it also strips out tracking parameters from the query string. Otherwise users might be surprised when their `gclid` is removed.